content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md

```diff
@@ -0,0 +1,297 @@
+---
+reviewers:
+- jayunit100
+- jsturtevant
+- marosset
+- perithompson
+title: कुबेरनेट्स में Windows कंटेनरों को शेड्यूल करने के लिए गाइड
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं।
```

Comment:
Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों का एक बडा हिस्सा है‌ ।

- While gathan karna is very appropriate for the verbatim translation, saying that "it is a big part" as opposed to "it constitutes a big part" in Hindi especially since you have used the word संगठनों sounds more appropriate.